tests/third_party/cupy/indexing_tests/test_insert.py

```diff
@@ -255,7 +255,6 @@ def test_putmask_differnt_dtypes_mask(self, xp, dtype):
         }
     )
 )
-@pytest.mark.usefixtures("allow_fall_back_on_numpy")
 class TestFillDiagonal(unittest.TestCase):
```

Comment:
Please update `tests/third_party/cupy/indexing_tests/test_generate.py` file with the latest changes to include tests for impacted functions.
Also please check if we can enable the scope for public CI.